### PR TITLE
Test against different ActiveSupport versions on Travis

### PR DESCRIPTION
**Pull request by XrXr**
I used Appraisals to generate the Gemfiles. This is an
extra development dependency but I think it's worth it
for the added convenience.

Closes #13

### CODE REVIEW

**Comment by rafaelfranca** [comment]
> This is an
> extra development dependency but I think it's worth it
> for the added convenience.

I prefer not to. It is easy to support multiple gemfiles without Appraisals. You can keep the Gemfiles generated by the tool and use `BUNDLE_GEMFILE` environment variable to test against it.

**Comment by Edouard-chin** [comment]
Looks like tests are failing because we use `Bundler.root` in the project https://github.com/Shopify/deprecation_toolkit/blob/9237c8b9f53bc7e15faed8a31ccd02667d8e4c4d/lib/deprecation_toolkit/read_write_helper.rb#L47 (deprecations gets read/written at the root of the project). Since we have gemfile inside a `gemfiles` subdirectory it now breaks. We'd have to find a workaround for that.

**Comment by XrXr** [comment]
@Edouard-chin what do you think about putting the gemfiles under the root of the repo?

**Comment by Edouard-chin** [comment]
The best would have been to get the root of the project without relying on Bundler but I have tried and it's non trivial.

I'd prefer if we leave the gemfiles inside a gemfiles folder and create the same deprecations directory with a symlink to the actual deprecations files. It make things more explicit

File: gemfiles/test [hunk 1]
```diff
@@ -0,0 +1 @@
+../test
```
Comment (Edouard-chin) [inline gemfiles/test]: should we just symlink the `test/deprecations` folder ?

**Pull request merged**